bagitobjecttransfer/recordtransfer/forms/user_forms.py

```diff
@@ -8,15 +8,29 @@
 from recordtransfer.constants import (
     ID_CONFIRM_NEW_PASSWORD,
     ID_CURRENT_PASSWORD,
+    ID_FIRST_NAME,
     ID_GETS_NOTIFICATION_EMAILS,
+    ID_LAST_NAME,
     ID_NEW_PASSWORD,
 )
 from recordtransfer.models import User
 
+import re
+
 
 class UserProfileForm(forms.ModelForm):
     """Form for editing user profile."""
 
+    first_name = forms.CharField(
+        widget=forms.TextInput(attrs={"id": ID_FIRST_NAME}),
+        label=_("First Name"),
+        required=True,
+    )
+    last_name = forms.CharField(
+        widget=forms.TextInput(attrs={"id": ID_LAST_NAME}),
+        label=_("Last Name"),
+        required=True,
+    )
```

Comment:
Django has a built-in field type for regex validation:

```suggestion
    first_name = forms.RegexField(
        regex=UserProfileForm.NAME_PATTERN,
        error_messages={
            "required": _("First name is required."),
            "invalid": UserProfileForm.NAME_VALIDATION_MESSAGE,
        },
        widget=forms.TextInput(attrs={"id": ID_FIRST_NAME}),
        label=_("First Name"),
        required=True,
    )
    last_name = forms.RegexField(
        regex=UserProfileForm.NAME_PATTERN,
        error_messages={
            "required": _("Last name is required."),
            "invalid": UserProfileForm.NAME_VALIDATION_MESSAGE,
        },
        widget=forms.TextInput(attrs={"id": ID_LAST_NAME}),
        label=_("Last Name"),
        required=True,
    )
```

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/281/commits/70fde46c390930b972d9f5174fa95b75d94f5ee7